package.json

```diff
@@ -25,9 +25,9 @@
   "keywords": [
     "oauth2"
   ],
-  "main": "dist/bundles/okta-react.cjs.js",
+  "main": "dist/src",
```

Comment:
Does this change impact the esm export?  (I haven't sat down to learn rollup yet)